### ISSUE TITLE
Installed Ajenti panel, it starts but accessing it makes it stop.

### ISSUE THREAD

**Issue by nomoredarkknight** [ajenti/ajenti#863]
I am on CentOS 7. So when I go to https://x.x.x.x:8000, it doesn't load anything. 

If I disable ssl from the Ajenti config file, then it connects without ssl.

With SSL enabled however, I have checked the status and it says "Ajenti is not running" even though it was running before trying to visit https://x.x.x.x:8000. Here is what is says when I run "ajenti-panel -v":

```
[root@localhost ~]# ajenti-panel -v
26.01.2016 15:14 ajenti-panel:105              INFO  Using config file /etc/ajenti/config.json
26.01.2016 15:14 ajenti-panel:130              INFO  Ajenti starting in foreground
26.01.2016 15:14 registry.py:282               DEBUG registered 'sha512_crypt' handler: <class 'passlib.handlers.sha2_crypt.sha512_crypt'>
26.01.2016 15:14 compat.py:20                  DEBUG Popen: git describe --tags 2> /dev/null
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['lsb_release', '-sd']
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['uname', '-mrs']
26.01.2016 15:14 core.py:60                    INFO  Ajenti 1.2.22.13 running on platform: centos
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin ajenti_org
26.01.2016 15:14 __init__.py:286               DEBUG Preloading plugin dependency: main
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin main
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin ajenti_org
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin apache
26.01.2016 15:14 __init__.py:286               DEBUG Preloading plugin dependency: webserver_common
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin webserver_common
26.01.2016 15:14 __init__.py:286               DEBUG Preloading plugin dependency: services
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin services
26.01.2016 15:14 __init__.py:286               DEBUG Preloading plugin dependency: dashboard
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin dashboard
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin services
26.01.2016 15:14 sm_upstart.py:49              INFO  Disabling Upstart service manager: org.freedesktop.DBus.Error.ServiceUnknown: The name com.ubuntu.Upstart was not provided by any .service files
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'systemctl']
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin webserver_common
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin apache
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'httpd']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [apache] skipping due to BinaryDependency (httpd)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin apcups
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'apcaccess']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [apcups] skipping due to BinaryDependency (apcaccess)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin bind9
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'named']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [bind9] skipping due to BinaryDependency (named)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin codemirror
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin configurator
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin cron
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'crontab']
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin csf
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'csf']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [csf] skipping due to BinaryDependency (csf)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin ctdb
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'ctdb']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [ctdb] skipping due to BinaryDependency (ctdb)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin db_common
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin dhcpd
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'dhcpd']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [dhcpd] skipping due to BinaryDependency (dhcpd)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin exports
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'nfsstat']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [exports] skipping due to BinaryDependency (nfsstat)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin fail2ban
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'fail2ban-client']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [fail2ban] skipping due to BinaryDependency (fail2ban-client)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin fm
26.01.2016 15:14 __init__.py:286               DEBUG Preloading plugin dependency: tasks
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin tasks
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin fm
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin fstab
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin hddtemp
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'hddtemp']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [hddtemp] skipping due to BinaryDependency (hddtemp)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin hdparm
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'hdparm']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [hdparm] skipping due to BinaryDependency (hdparm)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin hosts
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin ipmi
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'ipmitool']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [ipmi] skipping due to BinaryDependency (ipmitool)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin iptables
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'ip6tables']
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin lm_sensors
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'sensors']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [lm_sensors] skipping due to BinaryDependency (sensors)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin logs
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin megaraid
26.01.2016 15:14 __init__.py:346               DEBUG  *** [megaraid] skipping due to FileDependency (/opt/MegaRAID/MegaCli/MegaCli)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin memcache
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'memcached']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [memcache] skipping due to BinaryDependency (memcached)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin mongo
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'mongod']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [mongo] skipping due to BinaryDependency (mongod)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin munin
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'munin-cron']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [munin] skipping due to BinaryDependency (munin-cron)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin mysql
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'mysql']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [mysql] skipping due to BinaryDependency (mysql)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin netatalk
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'afpd']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [netatalk] skipping due to BinaryDependency (afpd)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin network
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin nginx
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'nginx']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [nginx] skipping due to BinaryDependency (nginx)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin notepad
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin nsd
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'nsd']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [nsd] skipping due to BinaryDependency (nsd)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin ntpd
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'ntpd']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [ntpd] skipping due to BinaryDependency (ntpd)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin openvpn
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'openvpn']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [openvpn] skipping due to BinaryDependency (openvpn)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin packages
26.01.2016 15:14 __init__.py:286               DEBUG Preloading plugin dependency: terminal
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin terminal
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin packages
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin plugins
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin power
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'systemctl']
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'pm-hibernate']
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin psql
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'psql']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [psql] skipping due to BinaryDependency (psql)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin raid
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'mdadm']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [raid] skipping due to BinaryDependency (mdadm)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin resolv
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin resources
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin samba
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'smbd']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [samba] skipping due to BinaryDependency (smbd)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin scripts
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin sensors
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin smartctl
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'smartctl']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [smartctl] skipping due to BinaryDependency (smartctl)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin snmpd
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'snmpd']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [snmpd] skipping due to BinaryDependency (snmpd)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin squid
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'squid3']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [squid] skipping due to BinaryDependency (squid3)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin supervisor
26.01.2016 15:14 compat.py:20                  DEBUG Popen: ['which', 'supervisord']
26.01.2016 15:14 __init__.py:346               DEBUG  *** [supervisor] skipping due to BinaryDependency (supervisord)
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin taskmgr
26.01.2016 15:14 __init__.py:297               DEBUG Loading plugin users
26.01.2016 15:14 __init__.py:207               DEBUG Root context instantiated [Inflater] <ajenti.ui.inflater.Inflater object at 0x1daab50>
26.01.2016 15:14 __init__.py:207               DEBUG Root context instantiated [UI] <ajenti.ui.UI object at 0x21b4850>
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout main:file-dialog-save
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout main:file-dialog
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout main:input-dialog
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout main:passwd-main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout ajenti_org:config
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout ajenti_org:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout dashboard:dash
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout dashboard:text
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout dashboard:welcome
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout services:bar
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout services:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout services:widget
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout webserver_common:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout configurator:ad-sync-config
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout configurator:ldap-sync-config
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout configurator:licensing
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout configurator:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout cron:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout db_common:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout tasks:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout tasks:params-copydir
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout tasks:params-deletedir
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout tasks:params-execute
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout tasks:params-rsync
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout fm:config
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout fm:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout fstab:free-widget
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout fstab:iio-widget
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout fstab:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout fstab:widget
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout hosts:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout iptables:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout iptables:option-address
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout iptables:option-custom
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout iptables:option-interface
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout iptables:option-match
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout iptables:option-port
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout iptables:option-ports
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout iptables:option-protocol
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout iptables:option-reject-with
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout iptables:option-state
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout iptables:option
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout logs:config
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout logs:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout network:bit-linux-basic
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout network:bit-linux-dhcp
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout network:bit-linux-ifupdown
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout network:bit-linux-ipv4
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout network:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout network:widget
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout notepad:config
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout notepad:listitem
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout notepad:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout terminal:config
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout terminal:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout packages:headers
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout packages:installer
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout packages:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout packages:row
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout plugins:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout power:widget
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout resolv:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout scripts:widget
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout sensors:cpu-line
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout sensors:cpu-widget
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout sensors:progressbar-widget
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout sensors:value-widget
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout taskmgr:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout users:main
26.01.2016 15:14 inflater.py:38                DEBUG Precaching layout users:user-list
26.01.2016 15:14 core.py:99                    INFO  Starting server on (u'0.0.0.0', 8000)
26.01.2016 15:14 __init__.py:207               DEBUG Root context instantiated [SessionMiddleware] <ajenti.middleware.SessionMiddleware object at 0x217db90>
26.01.2016 15:14 __init__.py:207               DEBUG Root context instantiated [AuthenticationMiddleware] <ajenti.middleware.AuthenticationMiddleware object at 0x21d0150>
26.01.2016 15:14 __init__.py:207               DEBUG Root context instantiated [CentralDispatcher] <ajenti.routing.CentralDispatcher object at 0x21d0110>
26.01.2016 15:14 core.py:134                   INFO  SSL enabled: /etc/ajenti/ajenti.pem
26.01.2016 15:14 __init__.py:207               DEBUG Root context instantiated [IPCServer] <ajenti.ipc.IPCServer object at 0x23c3710>
26.01.2016 15:14 __init__.py:207               DEBUG Root context instantiated [Licensing] <ajenti.licensing.Licensing object at 0x23c3610>
26.01.2016 15:14 feedback.py:30                DEBUG Feedback >> ping ({'edition': 'vanilla', 'version': '1.2.22.13', 'os': 'centos', 'id': 194555})
26.01.2016 15:14 feedback.py:35                DEBUG Feedback << {"status": "ok"}
```

**Comment by Xefir** [comment]
This was fixed in recent version (see #882). Can you update, test and close the issue if it's fine ?

**Comment by ghost** [comment]
I apply #882 and now work fine in Centos7, thanks
